css/index.css

```diff
@@ -166,6 +166,11 @@ footer nav a {
     align-items: flex-end;
 }
 
+.link {
```

Comment:
Instead of assigning a class to the links, you could have used `.button-box a` as the selector